cmd/destroy.go

```diff
@@ -164,7 +164,7 @@ func destroyLab(ctx context.Context, c *clab.CLab) (err error) {
 		}
 	}
 
-	log.Info("Removing container entries from /etc/hosts file")
+	log.Info("Removing containerlab host entries from /etc/hosts file")
```

Comment:
this is prone to error. 
you might want to reference the const with the filename... maybe the log messages should even go into the function?!

Reply:
Yes, move plz into the clab hosts

Reply:
I decided not to do that
it involves more changes that I wanted, the deletion function is called during the creation as well